### DISCUSSION TITLE
Getting detailed error from failed triggers

### DISCUSSION THREAD

**Discussion by NixBiks** [supabase/supabase#7061] [Questions]
I love the postgres triggers - however I often run into some errors that are almost impossible to determine why they are raised.

Is it possible to see the error message somewhere? E.g. if I do `raise exception 'My custom error message'` inside a trigger function. Where do I see this error? The supabase client does not return it...

**Comment by GaryAustin1** [comment]
This might be related: https://github.com/supabase/gotrue/pull/404 at least for auth triggers.
You can also use `raise log 'string' ` to get info and debug using the database postgres log.

**Reply by NixBiks** [reply]
Yeah unfortunately that is not available when running locally so I've added a feature request in supabase cli [here](https://github.com/supabase/cli/issues/271). I'd much rather play around locally and locate the issue than doing it in the cloud
